ui/app/i18n/registration/locale_en.json

```diff
@@ -113,5 +113,10 @@
   "DELETE_LABEL": "Delete",
   "EDIT_LABEL": "Edit",
   "REGISTRATION_LABEL_RELATIONSHIP_EXISTS": "Given relationship already exists",
-  "PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER": "Phone Number"
+  "PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER": "Phone Number",
+  "REGISTRATION_LABEL_SIMILAR_PATIENT_FOUND":"Similar patient found",
+  "REGISTRATION_LABEL_SYSTEM_ID":"System ID",
+  "REGISTRATION_IDTYPE_2_ART_KEY":"ART",
+  "REGISTRATION_LABEL_National_ID":"National ID",
```

Comment:
@IipperMangesh Please use full caps for REGISTRATION_LABEL_**National**_ID

Reply:
Yes, Changed it's key for 'CNI'.